operations/src/main/java/edu/unc/lib/boxc/operations/api/order/MemberOrderHelper.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2008 The University of North Carolina at Chapel Hill
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.unc.lib.boxc.operations.api.order;
+
+import edu.unc.lib.boxc.model.api.ResourceType;
+import edu.unc.lib.boxc.model.api.ids.PID;
+
+/**
+ * Helpers for member order operations
+ *
+ * @author bbpennel
+ */
+public class MemberOrderHelper {
+    private MemberOrderHelper() {
+    }
+
+    /**
+     * @param resourceType resource type to test
+     * @return true if the supplied resourceType supposed member ordering
```

Comment:
i think you have typo here. "supposed" seems like it should be "supports" ? and right now the method explicitly tests for a Work, but this comment makes it seem more open